src/modules/mount/main.py

```diff
@@ -321,6 +327,11 @@ def run():
         return (_("Configuration Error"),
                 _("No partitions are defined for <pre>{!s}</pre> to use.").format("mount"))
 
+    # swap
+    swap_partitions = [p['device'] for p in partitions if p['fs'] == 'linuxswap' and p['claimed'] == False ]
+    if ( swap_partitions != [] ):
```

Comment:
I don't think this `if` is needed.  If it is empty, the for loop will be skipped.